doc/source/news.rst

```diff
@@ -5,6 +5,26 @@
 News
 ====
 
+.. _release-12-0-9:
+
+Release 12.0.9 - 2022-10-28
+---------------------------
+
+Improvements
+------------
+
+* [:doc:`reference/functions/escalate`] Added a document for the ``escalate()`` function.
+
```

Comment:
We need to add https://github.com/groonga/groonga/commit/58e49dd6940f80b703f9efc908a60df05ae4c130.

Reply:
Added